powermonitor/render.js

```diff
@@ -0,0 +1,105 @@
+const electron = require('electron');
+const postis = require('postis');
+const { ipcRenderer } = electron;
+
+const {
+    POWER_MONITOR_EVENTS_CHANNEL,
+    POWER_MONITOR_MESSAGE_NAME,
+    POWER_MONITOR_QUERIES_CHANNEL
+} = require('./constants');
+
+/**
+ * The channel we use to communicate with Jitsi Meet window.
+ */
+let _channel;
+
+/**
+ * The listener for query responses.
+ * @param _ - Not used.
+ * @param response - The response to send.
+ */
+let queriesChannelListener = function (_, response) {
```

Comment:
Why this function definition and the next one are different than all other function definitions in the PR?

Reply:
Cause these a the listener we want to remove later.